File: exp/UnityChan2DTutorial/Assets/ScriptsCSharp/LoadingScene/LoadingScene.cs
using UnityEngine;
using UnityEngine.UI;
using System.Collections;

public class LoadingScene : MonoBehaviour {

	public Text t_coin ;
	public Text t_score ;
	public Text t_life ;

	void Start () {
		LoadValue() ;
		Invoke("WaitAndGo", 3) ;
	}

	void WaitAndGo(){
		Application.LoadLevel(2) ;
	}

	void LoadValue(){
		t_coin.text = GameState.coins+"" ;
		t_score.text = GameState.score+"" ;
		t_life.text = GameState.life+"" ;
	}
}

File: exp/UnityChan2DTutorial/Assets/ScriptsJs/LoadingStage/LoadingScene.js
#pragma strict
var t_coin : UI.Text ;
var t_score : UI.Text ;
var t_life : UI.Text ;

function Start () {
  LoadValue() ;
  Invoke("WaitAndGo", 3) ;
}

function WaitAndGo(){
  Application.LoadLevel(2) ;
}

function LoadValue(){
  t_coin.text = GameState.coins+"" ;
  t_score.text = GameState.score+"" ;
  t_life.text = GameState.life+"" ;
}